.github/workflows/validate.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v8
```

Comment:
Still getting an error:

Error: Failed to run: Error: invalid version string 'v1.64.8', golangci-lint v1 is not supported by golangci-lint-action >= v7., Error: invalid version string 'v1.64.8', golangci-lint v1 is not supported by golangci-lint-action >= v7.